geoips/commandline/geoips_tree.py

```diff
@@ -23,6 +23,27 @@ class GeoipsTree(GeoipsExecutableCommand):
     name = "tree"
     command_classes = []
 
+    describe_sectors_outputted = False
+    list_sectors_outputted = False
+
+    @property
+    def cmd_aliases(self):
+        """List of aliases we don't want to include in the tree.
+
+        Without this attribute there will be an entry for each alias of a command name.
+        So for 'list' (or 'ls'), there will be two 'geoips list' entries which we don't
+        want.
+        """
+        if not hasattr(self, "_cmd_aliases"):
+            self._cmd_aliases = []
+            for cmd_name in self.alias_mapping:
+                for alias in self.alias_mapping[cmd_name]:
+                    # Aliases below are names of actual commands and we need to deal
+                    # with this using conditionals in 'print_tree'
+                    if alias not in ["sector"]:
```

Comment:
I'm not sure how I feel about the overlap between a command (sector) and an alias for an artifact (sectors interface -> sector alias.  Not sure if artifact is the right term here)

I think this is fine for now, but we may want to consider the potential ambiguity.

Reply:
I'm resolving this

Reply:
The main reason I see keeping ``sector`` as an alias is for the describe plugin command. For example, if we run ``geoips describe sector canada``, that makes more sense grammatically than ``geoips describe sectors canada``. Not sure how we should handle this one.